doc/src/build/sui-local-network.md

```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
 }
 ```
 
+## Connect Command Line Tool to Local Network
+
+```bash
+# If this is your first time creating a local network, create a new environment with alias `local` for local network
```

Comment:
If this is the first time they have ever used the client CLI, or the first time after a clean install, they are asked if they want to connect to a Full node, y/N. It uses Devnet by default, and there is no other info about environments. So they'd need to choose a full node and create an address, then set the env alias. I just installed the binaries, started the local network, then used the RPC URL when I configured the client CLI the first time

Reply:
Yeah feel free to add more clarification about this case